src/Helpers/functions.php

```diff
@@ -105,10 +105,10 @@ function pred(...$vars)
      * Get repository object for the given repository name
      * 
      * @param string $repository
-     * @return \HZ\Illuminate\Mongez\Repository\RepositoryInterface
+     * @return \HZ\Illuminate\Mongez\Repository\MongoDBRepositoryManager
```

Comment:
It should be the repository interface not mongodb so it could be working on both MYSQL and MongoDB